src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+
+import usersFromServer from '../../api/users';
+import { Todo, TypeError } from '../../types';
+import { getMaxNumberId } from '../../utils/getMaxNumberId';
+import { getUserById } from '../../utils/getUserById';
+
+type Props = {
+  onAddTodo: (newElementTodo: Todo) => void;
+  todos: Todo[];
+};
+
+export const TodoForm: React.FC<Props> = ({ onAddTodo, todos }) => {
+  const [titleError, setTitleError] = useState<string>(TypeError.DEFAULT);
```

Comment:
u have `enum` as type for it)
```suggestion
  const [titleError, setTitleError] = useState<TypeError>(TypeError.DEFAULT);
```